sample/src/main/res/values-v17/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="CalendarHedStyle" parent="TextAppearance.MaterialCalendarWidget.Header">
```

Comment:
Maybe a typo. Shouldn't be CalendarHeaderStyle?

Reply:
Yes , it was a typo. Now it's fixed.